### PR TITLE
small edits to national pie map code

### PR DESCRIPTION
**Pull request by lindsaycarr**
Needed for https://github.com/USGS-VIZLAB/water-use-15/issues/200. This creates this national pie map:

national_pies

### CODE REVIEW

File: water_use_stories/R/draw_utils.R [hunk 1]
```diff
@@ -20,26 +20,32 @@ plot_national_pies <- function(us_states, us_counties, us_dots, metadata, waterm
...
+    #  "Desha,AR","Lincoln,AR","Bolivar,MS","Crittenden,AR","Tunica,MS","Quitman,MS","Tallahatchie,MS","White,AR",
+    #  "Butler,MO","Stoddard,MO","New Madrid,MO","Dunklin,MO","Pemiscot,MO","Mississippi,MO","Scott,MO","Independence,AR",
+    #  "Leflore,MS","Sunflower,MS", "Humphreys,MS","Washington,MS", "Chicot,AR","Drew,AR","Ashley,AR","Phillips,AR","Coahoma,MS",
+    # ag california
+    #"Kern,CA","Tulare,CA","Kings,CA","Fresno,CA", #"Pinal,AZ","Maricopa,AZ","La Paz,AZ","Yuma,AZ","Imperial,CA","Riverside,CA",
+    #  "Madera,CA","Merced,CA","Stanislaus,CA","San Joaquin,CA","Sacramento,CA","Solano,CA","Yolo,CA","Sutter,CA",
+    #  "Colusa,CA","Glenn,CA","Yuba,CA","Butte,CA"
+  )
+  us_counties@data <- us_counties@data %>% 
+    mutate(cnty_ref = paste0(NAME,',', dataRetrieval::stateCdLookup(STATEFP))) %>%
```
Comment (jread-usgs) [inline water_use_stories/R/draw_utils.R]: do you want to just delete these since we aren't using it?

File: water_use_stories/R/draw_utils.R [hunk 2]
```diff
@@ -20,26 +20,32 @@ plot_national_pies <- function(us_states, us_counties, us_dots, metadata, waterm
...
+    mutate(is.feature = cnty_ref %in% agg_blip)
+  
+  if(any(us_counties@data$is.feature)) {
```
Comment (jread-usgs) [inline water_use_stories/R/draw_utils.R]: same thing here - I say we delete this and keep it simple.

File: water_use_stories/R/draw_utils.R (hunk 1 above)
Comment (lindsaycarr) [reply water_use_stories/R/draw_utils.R]: done

File: water_use_stories/R/draw_utils.R (hunk 2 above)
Comment (lindsaycarr) [reply water_use_stories/R/draw_utils.R]: and done

**Pull request merged**